src/Extension/ExtendableTrait.php

```diff
@@ -180,6 +180,7 @@ public function addDynamicProperty($dynamicName, $value = null)
 
         if (!property_exists($this, $dynamicName)) {
             $this->{$dynamicName} = $value;
+            $this->purgeable[] = $dynamicName;
```

Comment:
Add a third parameter (`$purge = false`) to this method signature and use that instead to retain the existing behavior. Please also document that third parameter in the docs.

Reply:
On second thought, how about we just add it to the purgeable property in the rainlab.translate PR?